CHANGELOG.md

```diff
@@ -4,6 +4,17 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [unreleased]
 
+- Breaking change: Rename `onClose` to `onAfterClose` in `CartRewardCard`.
+- Breaking change: `mediaQueries` don't exposed all methods.
+  You must use `exposedMethods` prop to list methods who must be exposed.
```

Comment:
Je propose "You have to use `exposedMethods` prop to list the methods you want to expose on the `mediaQueries` HOC."